src/Sylius/Behat/Element/Admin/Currency/FormElement.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Element\Admin\Currency;
+
+use Behat\Mink\Session;
+use FriendsOfBehat\SymfonyExtension\Mink\MinkParameters;
+use Sylius\Behat\Element\Admin\Crud\FormElement as BaseFormElement;
+use Sylius\Behat\Service\Helper\AutocompleteHelperInterface;
+
+final class FormElement extends BaseFormElement implements FormElementInterface
```

Comment:
As we only have a creation page, with no editing, this form element does not quite make sense, as there is no need to separate the common logic and all these methods could be placed on the creation page 🤔